### ISSUE TITLE
Webpack compile lambda's sequentially

### ISSUE THREAD

**Issue by marcelpanse** [serverless-heaven/serverless-webpack#536]
# This is a (Bug Report / Feature Proposal)

## Description
Deploying an individual lambda function in our project (including compiling/packaging/deploying) takes only 4-5seconds. We have around 40+ lamda's in our project. When we do a full serverless deploy of all function, serverless will launch the webpack compiler for all 40 functions at the same time. This results in 100% CPU usage for 13 minutes because all the threads cannabalize on each other (also my computer because unbearable slow because of the CPU hog).
If all 40 functions would be compiled sequentially it should take around 5*40 = 2,5 to 3 minutes to compile everything (10 times faster!).
What would really help is a simple config flag that launches all compile threads sequentially instead.

* If there is additional config how would it look
custom:
  webpack:
    webpackConfig: 'webpack.config.js'
    sequentially: true

**Comment by tysoncadenhead** [comment]
This would be a really nice feature to have. This issue is keeping me from being able to package my functions individually because it hogs the threads and crashes in a CI environment.

**Comment by tysoncadenhead** [comment]
It looks like this is being worked on: https://github.com/serverless-heaven/serverless-webpack/pull/570

**Comment by roni-frantchi** [comment]
Regardless of #570 this one would be nice to have -
Though instead of setting it to sequentially or not, I wish we could control how many functions should be packaged in parallel:

```
custom:
  webpack:
    webpackConfig: 'webpack.config.js'
    parallel: 2
```

**Comment by Wytrykus** [comment]
This would be great to have indeed!
Does anybody know what part of the code would need to be modified?
In my naive mind, it should be rather simple to do the identical process sequentially instead of in parallel, or not?

**Comment by davidvalgamedios** [comment]
At least in my current version (5.8.0) you can set the desired concurrency. I guess setting concurrency to 1 would make it sequential.

```
custom:
  webpack:
    concurrency: 2
```

Docs: https://github.com/serverless-heaven/serverless-webpack#individual-packaging-concurrency

**Comment by vicary** [comment]
This should be well resolved via concurrency, please reopen if the problem persists.
